addons/website/static/src/js/backend/button.js

```diff
@@ -40,5 +41,63 @@ var WidgetWebsiteButton = AbstractField.extend({
     },
 });
 
+var WidgetWebsiteButtonIcon = AbstractField.extend({
+    template: 'WidgetWebsiteButtonIcon',
+    fieldDependencies: _.extend({}, AbstractFieldBinary.prototype.fieldDependencies, {
+        website_url: {type: 'string'},
+    }),
+    events: {
+        'click': '_onClick',
+    },
+
+    /**
+    * Register some useful references for later use throughout the widget.
```

Comment:
Convention: `Registers` (we conjugate the verb for documentation and not for inline comments)

But in fact, the comment is quite useless, it is like documenting the constructor with "initialize some stuff". Remove it, only leave `@override`